gdnative-core/src/core_types/geom/transform2d.rs

```diff
@@ -88,6 +94,15 @@ impl Transform2D {
             .scaled(scale)
     }
 
+    /// Constructs the transform from a given scale, angle (in radians), and origin.
```

Comment:
Can you maybe document that this is different from `IDENTITY.scaled().rotated().translated()`?

Reply:
done!